api-client/src/utils/geoJson.ts

```diff
@@ -1,42 +1,72 @@
 import distance from '@turf/distance'
-import { Position } from 'geojson'
 
 import { LocationType, PoiModel } from '../index'
-import { PoiFeature, PoiFeatureCollection } from '../maps'
+import { GeoJsonPoi, PoiFeature, PoiFeatureCollection } from '../maps'
 
 export const embedInCollection = (features: PoiFeature[]): PoiFeatureCollection => ({
   type: 'FeatureCollection',
   features,
 })
 
+// 5 meters
+const maxDistanceForOverlap = 0.005
```

Comment:
I reduced this distance, as otherwise it is not clear what belongs together (e.g. Cafè Tür an Tür and Tür an Tür Digitalfabrik would become one location if we kept the previous value for close pois)